lib/rust/ensogl/core/src/display/scene/layer.rs

```diff
@@ -851,11 +851,22 @@ impl LayerDynamicShapeInstance {
 // === Sublayers ===
 // =================
 
+/// Unboxed callback.
```

Comment:
I'd write here doc about what the callback does, not what this line is syntactically.